src/test/java/com/connexta/search/index/IndexRepositoryITest.java

```diff
@@ -30,23 +38,27 @@
 import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
 
+/**
+ * TODO Update this to a component or unit test, and test {@link
+ * com.connexta.search.common.SearchManagerImpl} instead
```

Comment:
I don't agree with this TODO. The integration with solr and its configuration/schema should be done at the repository level. The SearchManagerImpl class should mock this interface out.
I think we have a miscommunication on what differentiates a component and integration test.

Reply:
I have updated the javadoc to remove the terms.
```java
/**
 * TODO Update this to a unit test or use {@link
 * org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest}/{@link
 * org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest} instead.
 */
```